components/forms/ErrorMeesge.tsx

```diff
@@ -0,0 +1,5 @@
+import { ComponentProps } from "react";
+
+export default function ErrorMessage({className, ...props}: ComponentProps<"span">) {
+  return <span className={`label-text text-error text-right ${className || ""}`} {...props} />
+}
```

Comment:
The file name has a typo

Reply:
Currently, this file is being used with empty children, e.g.

```
<ErrorMessage className="w-full">{errors[name]?.message && "Required"}</ErrorMessage>
```

We got to check whether `ErrorMessage` is empty either in here or the component where `ErrorMessage` is being used.